server/controllers/recipes.js

```diff
@@ -20,48 +32,50 @@ export default {
           { userId: id }
         ]
       },
-    }))
-      .then((RecipeFound) => {
-        if (RecipeFound) {
+    })
+      .then((recipeFound) => {
+        if (recipeFound) {
           return res.status(409).json({
             error: 'You cannot create same recipe twice'
           });
         }
         return Recipe
           .create({
-            recipeName: req.body.recipeName,
-            description: req.body.description,
-            imageUrl: req.body.imageUrl,
-            ingredients: req.body.ingredients,
-            preptime: req.body.preptime,
-            servings: req.body.servings,
-            userId: userDetail.id
+            recipeName,
+            description,
+            imageUrl,
+            ingredients,
+            preptime,
+            servings,
+            userId: id
           })
-          .then(newRecipe => res.status(201).json({
-            recipeName: newRecipe.recipeName,
-            description: newRecipe.description,
-            ingredients: newRecipe.ingredients,
-            imageUrl: newRecipe.imageUrl,
-            preptime: newRecipe.preptime,
-            servings: newRecipe.servings,
-            id: newRecipe.id,
-            createdAt: moment().calendar(),
-            userId: newRecipe.userId
-          }));
+          .then(newRecipe =>
+            res.status(201).json({
+              recipeName: newRecipe.recipeName,
+              description: newRecipe.description,
+              ingredients: newRecipe.ingredients,
+              imageUrl: newRecipe.imageUrl,
+              preptime: newRecipe.preptime,
+              servings: newRecipe.servings,
+              id: newRecipe.id,
+              userId: newRecipe.userId
+            }));
       }).catch((error) => {
         res.status(500).json({ error: error.message });
       });
   },
 
+  /**
+   * function listAllRecipes
+   * @param {any} req
+   * @param {any} res
+   * @returns {array} all recipes
+   */
   listAllRecipes(req, res) {
     const limitValue = req.query.limit || 8;
     const pageValue = (req.query.page - 1) || 0;
-    // const sort = req.query.sort === 'upvotes' ||
-    //   req.query.sort === 'id' ?
-    //   req.query.sort : 'id';
     const sort = req.query.sort ? req.query.sort : 'createdAt';
```

Comment:
I'm wondering what would happen if a consumer of your api sends an invalid `sort` query value?